entries/entry02.md

```diff
@@ -1,8 +1,41 @@
 # Entry 2
```

Comment:
Sources: 0
EDP: 0
Knowledge: 1
Skills: 0
Professionalism: 1

Use the rubric!

Reply:
Late: 1/2 credit